duckduckgo/Android
# Feature/app shortcuts

Task/Issue URL: https://app.asana.com/0/488551667048375/711442573187879
Tech Design URL: 
CC: 

**Description**:
Adds app shortcuts for "new tab" and "clear data"

**Steps to test this PR**:
1. Install on Android 7.1 or later (Nougat MR1 device API level 25) 
1. Long press on the app icon; verify there are two options and the icons look ok
1. Open the app and perform a search
1. Test that clicking the "new tab" app shortcut button opens a new tab
1. Test that clicking the "clear data" button performs the same action as the `FireDialog`

1. Install on an older device and verify nothing breaks


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
